src/lib/graphqlErrorHandler.js

```diff
@@ -1,14 +1,21 @@
 import raven from "raven"
 import { assign } from "lodash"
+import { error as log } from "lib/loggers"
+import { GraphQLTimeoutError } from "lib/graphqlTimeoutMiddleware"
 
 const blacklistHttpStatuses = [401, 403, 404]
 
-export const shouldLogError = originalError => {
-  if (originalError && originalError.statusCode) {
-    return (
-      originalError.statusCode < 500 &&
-      !blacklistHttpStatuses.includes(originalError.statusCode)
-    )
+export const shouldReportError = originalError => {
+  if (originalError) {
+    if (originalError.statusCode) {
```

Comment:
Hi from the future 👋.

I found, after some digging, that this no longer works to identify and exclude `4xx` errors from Sentry. It likely worked at the time, but other changes in the meantime would require us to refer to the originating error this way (if we wanted to match 404s, e.g.):

    originalError.errors[0].originalError.statusCode  // 404

That's a little awkward. Would appreciate any help tracking down the cause or fixing this.